analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RedundantDeclaration.CSharp9.cs

```diff
@@ -10,6 +10,10 @@
 Record Local() => new Record(); // Compliant - FN
 static Record LocalStatic(Record r) => r;
 
+int? xx = ((new int?(5))); // Noncompliant {{Remove the explicit nullable type creation; it is redundant.}}
+
+var myEvent = new EventHandler((a, b) => { });
```

Comment:
Shouldn't this be C# 9 alternative?
```suggestion
EventHandler myEvent = new ((a, b) => { });
```